packages/x-data-grid-pro/src/components/GridDetailPanelToggleCell.tsx

```diff
@@ -19,8 +29,17 @@ const useUtilityClasses = (ownerState: OwnerState) => {
   return composeClasses(slots, getDataGridUtilityClass, classes);
 };
 
+const isExpandedSelector = createSelectorMemoized(
+  gridDetailPanelExpandedRowIdsSelector,
+  (expandedRowIds, rowId: GridRowId) => {
+    return expandedRowIds.has(rowId);
+  },
+);
```

Comment:
Now that we use a `Set` for selection state, we can replace `createSelectorMemoized` with `createSelector`.